app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -230,12 +230,8 @@ public void onCreate() {
                 .setOnPreferenceChangeListener(
                         (preference, newValue) -> {
                             Intent i = new Intent(activity, MainActivity.class);
```

Comment:
Why does this start the main activity and not the preferences? With `overridePendingTransition(0, 0)`, this could instantly switch the theme without the user being disrupted

Reply:
Oh I see...